projects/gdal/build.sh

```diff
@@ -46,9 +67,17 @@ cd ../..
 # build gdal
 cd gdal
 export LDFLAGS=${CXXFLAGS}
-./configure --without-libtool --with-liblzma --with-expat --with-sqlite3 --with-xerces --with-webp --with-netcdf=$SRC/install --with-curl=$SRC/install --without-hdf5 --with-jpeg=internal --with-proj=$SRC/install
+./configure --without-libtool --with-liblzma --with-expat --with-sqlite3 --with-xerces --with-webp --with-netcdf=$SRC/install --with-curl=$SRC/install/bin/curl-config --without-hdf5 --with-jpeg=internal --with-proj=$SRC/install
 make clean -s
-make -j$(nproc) -s static-lib
+make -j$(nproc) -s static-lib &
```

Comment:
lets remove & and next block of code. we can't do travis here due to long build, so can stop worrying on it as long as things work locally.

Reply:
I've removed that commit